network/kademlia.go

```diff
@@ -872,7 +996,7 @@ func (k *Kademlia) connectedNeighbours(peers [][]byte) (got bool, n int, missing
 	// in order deepest to shallowest compared to the kademlia base address
 	// all bins (except self) are included (0 <= bin <= 255)
 	depth := depthForPot(k.conns, k.NeighbourhoodSize, k.base)
-	k.eachConn(nil, 255, func(p *Peer, po int) bool {
+	k.eachConn(nil, nil, 255, func(p *Peer, po int) bool {
```

Comment:
Sames as above, maybe replace with `EachConn`

Reply:
Less cryptic is better.

We use lowercase method because the calling code is responsible for the lock.